src/sentry/static/sentry/app/views/performance/traceDetails/styles.tsx

```diff
@@ -73,3 +73,9 @@ export const StyledIconChevron = styled(IconChevron)`
   width: 7px;
   margin-left: ${space(0.25)};
 `;
+
+export const TransactionBarTitleContent = styled('span')`
+  display: flex;
+  align-items: center;
+  gap: ${space(0.75)};
```

Comment:
I don't think using `gap` with flex is fully supported still, just do a margin on one of the items.

Reply:
Good call, looks like it's not supported in safari [link](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#browser_compatibility). Fixed in f8b82b0